### PR TITLE
remove trusted types on empty string

### PR DESCRIPTION
**Pull request by ruberino**
- Removed trustedTypes because this created issues in HN-Skjema when rendering empty translations in sidebar

### CODE REVIEW

**Comment by andreasnp** [comment]
Bra jobba Ruben 😊

File: src/util/index.ts [hunk 1]
```diff
@@ -150,14 +150,14 @@ export function getSublabelText(
   resources?: Resources
 ): string {
   if (item) {
-    const markdown = getSublabelExtensionValue(item) || (window.trustedTypes ? (window.trustedTypes.emptyHTML as unknown as string) : '');
+    const markdown = getSublabelExtensionValue(item) || '';
     return markdown
       ? getMarkdownValue(markdown, item, onRenderMarkdown, questionnaire, resources?.linkOpensInNewTab)
       : window.trustedTypes
       ? (window.trustedTypes.emptyHTML as unknown as string)
```
Comment (andreasnp) [inline src/util/index.ts:157]: Fjerne 'as unknown as string' her også?

**Review by andreasnp — COMMENTED** [review]
Dette likte jeg 😊

**Review by AG-83 — APPROVED** [review]
Bra jobba :D

**Pull request merged**